pkg/grizzly/server.go

```diff
@@ -383,6 +386,78 @@ func (s *Server) RootHandler(w http.ResponseWriter, _ *http.Request) {
 	}
 }
 
+func (s *Server) NewResourceGetHandler(w http.ResponseWriter, r *http.Request) {
+	isaFile, err := isFile(s.ResourcePath)
+	if err != nil {
+		SendError(w, "Error finding resource path", err, 400)
```

Comment:
Its better to have different error messages. Maybe something like this?

```suggestion
		SendError(w, "Resource path is not a file", err, 400)
```